README.md

```diff
@@ -11,15 +11,24 @@ make sure that the firmware in your module is compatible with Mbed OS, follow th
 It is advisable to update the [AT firmware](https://www.espressif.com/en/support/download/at?keys=) at least to version
 1.7.0.0
 
+v1.6 driver supports AT firmware up to version 1.6.2. Mbed OS 5.10 supports AT firmwares up to version 1.6.2. v1.7
```

Comment:
Remember that this is going to be inside Mbed OS.
The driver version tags are not going to be visible for the user, so I would remove all references to driver version numbers.

Just tell that THIS driver supports AT firmware versions from x.x.x to y.y.y and we recommend z.z.z.

Then whenever it goes out in Mbed OS, users realise that Mbed OS 5.9 did not have this mentioned but 5.11 does, so that is when the "driver" started to support nevest driver..

Reply:
ok, to be fixed.